### ISSUE TITLE
Mettre en forme la div info film

### ISSUE THREAD

**Issue by RyderBlack** [eukavlin/movie-data#14]
@eukavlin soit on garde la forme actuelle sinon tu peux faire des changements. 
Mais le plus urgent c'est de styliser la div info pour qu elle soit plus présentable.

**Comment by eukavlin** [comment]
Oui, le + urgent en premier, a savoir la div info. Pour le reste, j'ai une idée, mais c'est que du bonus pour le background ;P.
